modules/utxo-ord/src/inscriptions.ts

```diff
@@ -0,0 +1,65 @@
+/*
+Functions for dealing with inscriptions.
+
+See https://docs.ordinals.com/inscriptions.html
+*/
+
+import * as assert from 'assert';
+import { Payment, script as bscript } from 'bitcoinjs-lib';
```

Comment:
You can also import these from `utxo-lib` - it reexports everything from `bitcoinjs-lib`.

Reply:
ah didnt even have the bitcoinjs-lib module imported. Updated it import from utxo-lib instead since we have this dep already.